annotation/tests/test_tasks_services.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from sqlalchemy.orm import Session
+
+from annotation.errors import FieldConstraintError
+from annotation.jobs.services import ValidationSchema
+from annotation.tasks.services import validate_task_info
+
+
+@pytest.fixture
+def db_session():
+    """Fixture for creating a mock database session."""
+    return Mock(spec=Session)
```

Comment:
check out teams chat regarding fixtures

Reply:
use patch for this. If you want just a mock object, then no need for this fixture, move it into test

Reply:
and also use yield instead of return

Reply:
Fixed :)